data_structures/reverse_a_linked_list.cpp

```diff
@@ -105,7 +123,7 @@ void list::insert(int32_t n) {
  * @returns void
  */
 void list::reverseList() {
-    Node *curr = head;
+    Node* curr = head;
     Node *prev = nullptr, *next_node = nullptr;
```

Comment:
Could you make the pointer placement consistent? Some of the places you're using `* identifier = something` and other places the ` *identifier = something`...

Reply:
This was done by `clang-format`. I guess the mentioned inconsistency helps to avoid the following _trap_: how many pointers are defined in the line below:
```cpp
int* x, y;
```

In any case, please have a look at 4cddaf83a633fe58a4aa89e7a50810037ccaef83.